### PR TITLE
added SpanContaining query

### LINKED ISSUE #97: Check if there are any new queries or aggs that we still do not support.

### PR DESCRIPTION
**Pull request by einorler**
Part of #97

### CODE REVIEW

File: src/Query/Span/SpanContainingQuery.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ * Elasticsearch span containing query.
+ *
+ * @link https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-span-containing-query.html
+ */
+class SpanContainingQuery implements SpanQueryInterface
+{
+    /**
+     * @param SpanQueryInterface
+     */
+    private $big;
```
Comment (saimaz) [inline src/Query/Span/SpanContainingQuery.php]: Missing setters and getters for big and little.

**Comment by einorler** [comment]
Blocks #101

**Comment by saimaz** [comment]
Span containing is missing parameters trait.

File: src/Query/Span/SpanContainingQuery.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    {
+        return 'span_containing';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function toArray()
+    {
+        $out = [
```
Comment (saimaz) [inline src/Query/Span/SpanContainingQuery.php]: Every single query has `$output` variable. Please keep consistency.

**Comment by einorler** [comment]
Included changes suggested in the last comment

**Pull request merged**